.github/workflows/scheduled_e2e.yml

```diff
@@ -40,10 +40,24 @@ jobs:
           sa-api-token: ${{secrets.E2E_SERVICE_ACCOUNT_API_TOKEN}}
           user-api-token: ${{secrets.E2E_NEPTUNE_API_TOKEN}}
 
-  notify:
+  publish-tests:
     needs: [test-e2e]
     runs-on: ubuntu-latest
     if: always()
+    permissions:
+      checks: write
+      pull-requests: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Publish tests
+        uses: ./.github/actions/publish-tests-result
+
+  notify:
+    needs: [test-e2e]
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
```

Comment:
`github.ref == 'refs/heads/master'` github.ref, because we sometimes use this workflow in branch to verify e2e